workspace/src/utility.js

```diff
@@ -14,11 +14,11 @@ export function makeOpDel (pos, length) {
   };
 };
 
-export function makeOpMov (pos, value, des) {
+export function makeOpMov (pos, length, des) {
   return {
     type : "mov",
     pos : pos,
-    value : value,
+    value : length,
```

Comment:
The problem is not length. The problem is that you are still using `value` as the key.